### PR TITLE
🌐 Add Japanese translation for `docs/ja/docs/tutorial/dependencies/index.md`

### PR DESCRIPTION
**Pull request by SwftAlpc**
This PR translates the tutorial/dependencies/index.md to Japanese.
Please review tutorial/dependencies/index.md.

~Issue #1571 generates a new directory to support Japanese.
PR #1656 generates a new directory for tutorial.
I will fix dependency after #1571 & #1656 are merged.~

### CODE REVIEW

**Review by tokusumi — CHANGES_REQUESTED** [review]
thank you for translating difficult chapter!

**Comment by SwftAlpc** [comment]
@tokusumi 
I approved your suggetions! Thank you!

**Review by tokusumi — APPROVED** [review]
LGTM 🚀

File: docs/ja/docs/tutorial/dependencies/index.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+この方法では、共有されるコードを一度書き、**FastAPI** が*path operations*のための呼び出しを行います。
+
+!!! check "確認"
+    特別なクラスを作成してどこかで **FastAPI** に渡して「登録」する必要はないことに注意してください。
+
+    `Depends`を渡すだけで、**FastAPI** が残りの処理をしてくれます。
+
+## `async`にするかどうか
+
+依存関係は **FastAPI**（*path operation関数*と同じ）からも呼び出されるため、関数を定義する際にも同じルールが適用されます。
```
Comment (sh0nk) [inline docs/ja/docs/tutorial/dependencies/index.md:107]:
```suggestion
依存関係も（*path operation関数*と同様に） **FastAPI** から呼び出されるため、関数を定義する際にも同じルールが適用されます。
```
To clarify the meaning of the sentence.

File: docs/ja/docs/tutorial/dependencies/index.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+* リソース
+* プロバイダ
+* サービス
+* インジェクタブル
+* コンポーネント
+
+## **FastAPI** プラグイン
+
+統合や「プラグイン」は **依存性注入** システムを使って構築することができます。しかし、実際には、**「プラグイン」を作成する必要はありません**。依存関係を使用することで、無限の数の統合やインタラクションを宣言することができ、それが**path operation関数*で利用可能になるからです。
```
Comment (sh0nk) [inline docs/ja/docs/tutorial/dependencies/index.md:146]:
```suggestion
統合や「プラグイン」は **依存性注入** システムを使って構築することができます。しかし、実際には、**「プラグイン」を作成する必要はありません**。依存関係を使用することで、無限の数の統合やインタラクションを宣言することができ、それが*path operation関数*で利用可能になるからです。
```
typo

File: docs/ja/docs/tutorial/dependencies/index.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+また、通常の`def`*path operation関数*の中に`async def`を入れて依存関係を宣言したり、`async def`*path operation関数*の中に`def`を入れて依存関係を宣言したりすることなどができます。
+
+それは重要ではありません。**FastAPI** は何をすべきかを知っています。
+
+!!! note "備考"
+    わからない場合は、ドキュメントの[Async: *"In a hurry?"*](../../async.md){.internal-link target=_blank}の中の`async`と`await`についてのセクションを確認してください。
+
+## OpenAPIとの統合
+
+依存関係（およびサブ依存関係）のすべてのリクエスト宣言、検証、および要件は、同じOpenAPIスキーマに統合されます。
```
Comment (sh0nk) [inline docs/ja/docs/tutorial/dependencies/index.md:120]:
```suggestion
依存関係（およびサブ依存関係）のすべてのリクエスト宣言、バリデーション、および要件は、同じOpenAPIスキーマに統合されます。
```
To be consistent with #L207. (The other way round is fine too.)

**Review by sh0nk — COMMENTED** [review]
Let me add just a couple of suggestions against typo and nitpicks!

**Comment by tiangolo** [comment]
📝 Docs preview for commit fcb10ad3cc7489530f7343338c38e1cf0471ad4a at: https://6499cb4e7bcac500837f92f0--fastapi.netlify.app

**Comment by tiangolo** [comment]
As this was handled in https://github.com/tiangolo/fastapi/pull/1958, I'll go ahead and close this one. Thank you! ☕

**Pull request closed without merging**